grails-app/domain/org/broadinstitute/orsp/IssueExtraProperty.groovy

```diff
@@ -63,6 +69,7 @@ class IssueExtraProperty {
     String name
     String value
     String projectKey
+    List<IssueExtraProperty> values
```

Comment:
I'm not a fan of the idea of nesting this object on itself. Can we look into alternatives to this approach?